cdap-source-control/src/test/java/io/cdap/cdap/sourcecontrol/RepositoryManagerTest.java

```diff
@@ -114,11 +116,11 @@ public void testValidateInvalidToken() throws Exception {
   public void testValidateInvalidBranchName() {
     String serverURL = gitServer.getServerURL();
     RepositoryConfig config = new RepositoryConfig.Builder().setProvider(Provider.GITHUB)
-                                                            .setLink(serverURL + "ignored")
-                                                            .setDefaultBranch("develop-invalid")
-                                                            .setAuthType(AuthType.PAT)
-                                                            .setTokenName(GITHUB_TOKEN_NAME)
-                                                            .build();
+      .setLink(serverURL + "ignored")
```

Comment:
can move to separate function that sets RepositoryConfig as it is re-used?

Reply:
It is being done in another PR that was raised before this, but is under review. It should be merged soon: https://github.com/cdapio/cdap/pull/14917/files